docs/components/DocumentationHeader.tsx

```diff
@@ -168,7 +168,21 @@ const SECTION_LINK_TEXT = css`
   right: 0;
 `;
 
-function SectionContainer({ spaceBetween = 0, spaceAround = 0, children, style, className }) {
+type SectionContainerProps = {
+  children: JSX.Element[];
```

Comment:
Unfortunately not, because default children from `React.FC` is of type `React.ReactNode` which is sth like `string | null | JSX.Element | JSX.ElementArray | ...`, and we need here only children of that specified type, because we iterate through them.